examples/montage/index.html

```diff
@@ -2,25 +2,28 @@
 <html lang="en" data-framework="montage" id="montage-todomvc">
 	<head>
 		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Why did you added this? https://github.com/tastejs/todomvc/commit/9429daa2d4ce051c8e95a34d4244b0e100e6c803